tests/test_resolutions.py

```diff
@@ -243,7 +249,7 @@ def test_description(self) -> None:
 
         ctt = ContainsTheText(test_text)
 
-        expected_description = f'Containing the text "{test_text}".'
+        expected_description = "Containing the text 'Wenn ist das Nunstück git und Slotermeyer?'."
```

Comment:
Oh no, now the people reading the test will read half of the funniest joke in the world! They could be rendered helpless in fits of giggles for at least an hour!